source/sessions/tests/dirty-session-errors.yml

```diff
@@ -135,10 +140,9 @@ tests:
           session: session0
       - name: endSession
         object: session0
-      - name: assertServerSessionPoolSize
+      - *find_with_implicit_session
```

Comment:
This diff also removes the `insertOne` operation.

Side question, if you're repeating two forms of inserts (w/ and w/o explicit sessions), would it make sense to create anchors for them as well as you did for `find_with_implicit_session`?

Reply:
The insertOne operations are still there.

The latest commit adds anchors for `insert_with_explicit_session`/`insert_with_implicit_session`.

Reply:
Ok, I must have misread the diff.